### ISSUE TITLE
users/{id}.json endpoint missing

### ISSUE THREAD

**Issue by mrosenberg** [tick/tick-api#42]
We have a script that runs on a daily cron to query the tick api. The script asks for all updated entries from the previous day and stores those in a relational db with foreign keys for user and task. If a downloaded entry's foreign key fails we fetch the task at tasks/{id}.json. That endpoint doesn't exist for users. Instead we are forced to download all of the users every time we add someone to tick. 

Having the individual endpoint would be very helpful.

**Comment by johnlpollard** [comment]
Thanks @mrosenberg! I will share it with the team.

**Comment by tapanparekh** [comment]
@johnlpollard 
Any update on this?
Is this implemented?

**Comment by johnloringpollard** [comment]
Can you try a GET with users/{id}.json now? It should be an available end point now.

**Comment by tapanparekh** [comment]
> Can you try a GET with users/{id}.json now? It should be an available end point now.

Yes, I am getting user detail by users/{id}.json now. Thank you.

**Comment by tapanparekh** [comment]
As I am getting user detail by user id, the response should be the a user object. Currently I am getting array.

**Comment by johnloringpollard** [comment]
Can you check now?

**Comment by tapanparekh** [comment]
> Can you check now?

{
    "status": 500,
    "error": "Internal Server Error"
}

**Comment by tapanparekh** [comment]
There is no error now, working good, Thank you so much for your quick support.
